api/kyverno/v1/image_verification_types.go

```diff
@@ -249,6 +264,25 @@ type Attestation struct {
 	Conditions []AnyAllConditions `json:"conditions,omitempty" yaml:"conditions,omitempty"`
 }
 
+type ImageRegistryCredentials struct {
+	// Helpers specifies a list of OCI Registry names, whose authentication helpers are provided
+	// It can be of one of these values: AWS, Azure, GCP, GHCR
+	// +kubebuilder:validation:Optional
+	Helpers []ImageRegistryCredentialsHelpersType `json:"helpers,omitempty" yaml:"helpers,omitempty"`
+
+	// Secrets specifies a list of secrets that are provided for credentials
+	// Secrets must be in Kyverno namespace
+	// +kubebuilder:validation:Optional
+	Secrets []Secret `json:"secrets,omitempty" yaml:"secrets,omitempty"`
```

Comment:
Aren't secrets per credential helper ? cc @JimBugwadia

Reply:
`Secrets` are equivalent to the input in `imagePullSecrets` flag